### PR TITLE
Tune management callflags for aspidochelone

### PR DESCRIPTION
**Pull request by roman-khimov**
Not tested yet.

### CODE REVIEW

**Comment by roman-khimov** [comment]
Works like charm, mainnet/testnet compatibility is still there, neofs mainnet sidechain can be processed too.

File: pkg/core/native/interop.go [hunk 1]
```diff
@@ -38,9 +40,14 @@ func Call(ic *interop.Context) error {
 	if !ok {
 		return fmt.Errorf("method not found")
 	}
-	if !ic.VM.Context().GetCallFlags().Has(m.RequiredFlags) {
+	reqFlags := m.RequiredFlags
+	if !ic.IsHardforkEnabled(config.HFAspidochelone) && c.Metadata().ID == ManagementContractID &&
+		(m.MD.Name == "deploy" || m.MD.Name == "update") {
```
Comment (AnnaShaleva) [inline pkg/core/native/interop.go:46]: If we're doing this for both `deploy` and `update`, then maybe it's worth to add `deployWithData` and `updateWithData` also to preserve full consistency with the old behaviour? Even if `*WithData` are not related to this state diff.

**Comment by AnnaShaleva** [comment]
Otherwise LGTM.

File: pkg/core/native/interop.go (hunk 1 above)
Comment (roman-khimov) [reply pkg/core/native/interop.go:46]: It's `MD.Name ==`, so it works for any number of deploy/update parameters.

Comment (AnnaShaleva) [reply pkg/core/native/interop.go:46]: Oh, somehow I've looked at the method implementation's names, not at the method's names, sorry.

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**